tests/integration/api/relayer.rs

```diff
@@ -0,0 +1,266 @@
+
+    if networks.is_empty() {
+        panic!("No networks selected for testing");
+    }
+
+    let registry = TestRegistry::load().expect("Failed to load test registry");
+
+    let network = networks
+        .iter()
+        .find(|n| {
+            registry
+                .get_network(n)
+                .map(|c| c.network_type == "evm")
+                .unwrap_or(false)
+        })
+        .expect("No EVM network found in selection");
+
+    let network_config = registry
+        .get_network(network)
+        .expect("Network not found in registry");
+
+    info!(network = %network, "Testing relayer CRUD");
+
+    let client = RelayerClient::from_env().expect("Failed to create RelayerClient");
+
+    let cleanup_count = client
+        .delete_all_relayers_by_network(network)
+        .await
+        .expect("Failed to clean up existing relayers");
+
+    if cleanup_count > 0 {
+        debug!(
+            count = cleanup_count,
+            network = %network,
+            "Cleaned up existing relayers"
+        );
+    }
+
+    let relayer_id = uuid::Uuid::new_v4().to_string();
+    let create_request = CreateRelayerRequest {
+        id: Some(relayer_id.clone()),
+        name: format!("Test CRUD Relayer - {}", network),
+        network: network.to_string(),
+        paused: false,
+        network_type: RelayerNetworkType::Evm,
+        policies: None,
+        signer_id: network_config.signer.id.clone(),
+        notification_id: None,
+        custom_rpc_urls: None,
+    };
+
+    let created = client
+        .create_relayer(create_request)
+        .await
+        .expect("Failed to create relayer");
+
+    info!(relayer_id = %created.id, "Created relayer");
+    assert_eq!(created.id, relayer_id);
+    assert_eq!(created.network, network.to_string());
+    assert_eq!(created.network_type, RelayerNetworkType::Evm);
+    assert!(!created.paused);
+
+    let fetched = client
+        .get_relayer(&relayer_id)
+        .await
+        .expect("Failed to get relayer");
+
+    info!(
+        relayer_id = %fetched.id,
+        name = %fetched.name,
+        network = %fetched.network,
+        address = ?fetched.address,
+        "Fetched relayer"
+    );
+    debug!(
+        system_disabled = ?fetched.system_disabled,
+        disabled_reason = ?fetched.disabled_reason,
+        "Relayer status details"
+    );
+
+    assert_eq!(fetched.id, relayer_id);
+    assert_eq!(fetched.network, network.to_string());
+    assert_eq!(fetched.signer_id, network_config.signer.id);
+    assert!(fetched.address.is_some(), "Relayer should have an address");
+
+    client
+        .delete_relayer(&relayer_id)
+        .await
+        .expect("Failed to delete relayer");
+    info!(relayer_id = %relayer_id, "Deleted relayer");
+
+    let result = client.get_relayer(&relayer_id).await;
+    assert!(result.is_err(), "Relayer should not exist after deletion");
+
+    info!("Relayer CRUD test passed");
```

Comment:
We have asserts so we would definitely know if they are passing or not. Logs are not needed.